### ISSUE TITLE
Unicode character 11/(\v)/vertical tab not being rendered properly

### ISSUE THREAD

**Issue by chiefnoah** [gambit/gambit#825]
It looks like vertical tab (`\v`) doesn't get rendered properly in a `(display)`.

Example:
```scheme
Gambit v4.9.4-180-g266c81f3

> (display "Hello\vWorld!\n")
HelloWorld!
> 
```
`gsi` compiled from `HEAD` of `master` as of today.

C equivalent:

```c
#include <stdio.h>

int main(void) {
    printf("Hello\vWorld!");
}
```
Output:
```
Hello
     World!
```

`\v` is explicitly documented as supported in the string escape section of the docs.

**Comment by gambiteer** [comment]
The REPL may not be doing what you think it should do, but "display" seems to work fine.
If I
```
> (with-output-to-file "crap14" (lambda () (display "Hello\vWorld!\n")))
```
then
```
[Bradleys-Mac-mini:~] lucier% cat crap14
Hello
     World!
```

**Comment by chiefnoah** [comment]
Oh interesting! I did pipe it into a vim buffer and it looks like it's rendering the `^K` in both the C and the Scheme implementations, but Scheme has an extra `\n` in the output... Huh? It's probably my terminal then.

**Comment by gambiteer** [comment]
I don't see an extra `\n` at the end of `crap14`.

**Comment by feeley** [comment]
Thanks for your report! Actually this is a bug in the terminal emulation layer (implemented by lib/os_tty.c). Now fixed by 77c2da51.  For portability of output to the terminal (i.e. same behaviour on Windows, linux, macOS) the Gambit runtime system interprets the stream of bytes destined to the terminal (to check for escape sequences and special control characters) and performs the operations requested, which can be rendered as a different sequence of bytes that are actually sent to the terminal (depending on the tty's termcap). VTAB was not handled. Note that sending the output of the `display` to a file bypasses the terminal emulation layer, so the bytes are untouched. You can also bypass the terminal emulation for terminal output with the `-:tE` runtime option on the command line.
